### DISCUSSION TITLE
Cyber Desk on Wio Terminal using Platform IO, powered by Cursor AI | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1994] [Q&A]
# Cyber Desk on Wio Terminal using Platform IO, powered by Cursor AI | Seeed Studio Wiki

- The Whole Project Overview

https://wiki.seeedstudio.com/CyberDeck_Wio-Terminal/

**Comment by rojecas** [comment]
Unusable code example, the library version linked (@2.1.0) does not exist, it only was published until 1.6.0

**Reply by nfs0619** [reply]
Thanks for reaching out! For assistance with the library version issue, we recommend contacting our technical support team at techsupport@seeed.io. They can provide answers to your questions and help resolve this issue.
